cloudferry_devlab/cloudferry_devlab/tests/config.py

```diff
@@ -379,7 +396,10 @@
           'nics': [{'net-id': 'tenant5net2', 'v4-fixed-ip': '123.2.2.101'}]}
          ],
      'images': [{'name': 'cirros_image_for_tenant5', 'copy_from': img_url,
-                 'is_public': True}],
+                 'is_public': True}]
+     },
+    {'name': 'tenant6', 'description': 'None',
+     'enabled': True, 'capital': True
```

Comment:
Also please add docstring with description of a test related to that tenant name